Src/Microsoft.Dynamic/Debugging/CollectionUtils.cs

```diff
@@ -29,17 +29,22 @@ internal static class CollectionUtils {
             if (first.Count != second.Count) {
                 return false;
             }
+
             var cmp = EqualityComparer<T>.Default;
-            var f = first.GetEnumerator();
-            var s = second.GetEnumerator();
-            while (f.MoveNext()) {
-                s.MoveNext();
 
-                if (!cmp.Equals(f.Current, s.Current)) {
-                    return false;
+            using (var f = first.GetEnumerator()) {
```

Comment:
You can get rid of the extra indent/brackets:
```C#
using (var f = first.GetEnumerator())
using (var s = second.GetEnumerator()) {
        // ...
}
```

Reply:
I would prefer to keep the brackets if you don't mind.